docs/Deploying-Pega-on-AKS.md

```diff
@@ -509,7 +511,7 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
+    | stream.enabled: stream.bootstrapServer: stream.securityProtocol: stream.trustStore: stream.trustStorePassword: stream.keyStore: stream.keyStorePassword: stream.saslMechanism: stream.jaasConfig: stream.streamNamePattern: stream.replicationFactor: stream.external_secret_name:| Either to enable stream with external kafka and configure the required settings | <ul><li>enabled: true/false <br/> Set to true if you want to deploy pega platform with external kafka, otherwise false. *Note: Pega recommends external kafka because stream tier is deprecated since 8.7.*</li><li>bootstrapServer: Provide External kafka broker urls(comma separated) </li><li>securityProtocol: Provide Security Protocol used to communicate with brokers. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL. </li><li> trustStore: If required, provide a Kubernetes secret name that holds the trustStore certificate. Please add the Kubernetes secret name in 'global.certificatesSecrets' too. Pega supports only Java Key Store (.jks) format for keystore.</li><li> trustStorePassword: If required, provide keyStore value in plain text. </li><li> keyStore: If required, provide a Kubernetes secret name that holds the keyStore certificate. Please add the Kubernetes secret name in 'global.certificatesSecrets' too. Pega supports only Java Key Store (.jks) format for keystore.<li> keyStorePassword: If required, provide keyStore value in plain text. </li><li> jaasConfig: If required, provide jaasConfig value in plain text. </li><li> saslMechanism: If required, provide SASL Mechanism, Supported values are: PLAIN, SCRAM-SHA-256, SCRAM-SHA-512 </li> <li> streamNamePattern: By default, topics originating from Pega Platform have the pega- prefix, so that it is easy to distinguish them from topics created by other applications. You can configure the name pattern to customize topic names per environment </li> <li> replicationFactor: Value of replicationFactor cannot be more than number of brokers </li> <li> external_secret_name: To avoid exposing trustStorePassword, keyStorePassword, and jaasConfig parameters, leave the values empty and configure them in External Secrets Manager, and enter the external secret name here make sure the keys in the secret are STREAM_TRUSTSTORE_PASSWORD, STREAM_KEYSTORE_PASSWORD and STREAM_JAAS_CONFIG respectively. </li></ul> |
```

Comment:
I made a few context adjustments, 
  | stream.enabled: stream.bootstrapServer: stream.securityProtocol: stream.trustStore: stream.trustStorePassword: stream.keyStore: stream.keyStorePassword: stream.saslMechanism: stream.jaasConfig: stream.streamNamePattern: stream.replicationFactor: stream.external_secret_name:| Enable an externalized kafka configuration to connect to your existing stream service, by configuring these required settings | <ul><li>enabled: true/false <br/> Set to `true` if you want to deploy Pega Platform to use an externalized kafka configuration, otherwise leave set to `false`. *Note: Pega recommends enabling an externalized kafka configuration and has deprecated using a stream tier configuration starting at version 8.7.*</li><li>bootstrapServer: Provide your existing kafka broker URLs separated by commas.</li><li>securityProtocol: Provide the required security protocol your deployment will use to communicate with your existing brokers. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL. </li><li> trustStore: If required, provide a Kubernetes secret name that holds the trustStore certificate, and add the Kubernetes secret name using the `global.certificatesSecrets` parameter. Pega only supports using the Java Key Store (.jks) format for trustStores (**TAZ>>>NOT KEYSTORE, right????**).</li><li> trustStorePassword: If required, provide trustStore (**TAZ>>>NOT KEYSTORE, right????**)value in plain text. </li><li> keyStore: If required, provide a Kubernetes secret name that holds a keyStore certificate. Add the Kubernetes secret name using the `global.certificatesSecrets` parameter. Pega only supports the Java Key Store (.jks) format for keystores.<li> keyStorePassword: If required, provide a keyStore value in plain text. </li><li> jaasConfig: If required, provide jaasConfig value in plain text. </li><li> saslMechanism: If required, provide a SASL mechanism. Supported values are: PLAIN, SCRAM-SHA-256, SCRAM-SHA-512 </li> <li> streamNamePattern: By default, topics originating from Pega Platform have the "pega-" prefix, so that it is easy to distinguish them from topics created by other applications. Pega supports customizing the name pattern for your Externalized Kafka configuration for each deployment.</li> <li> replicationFactor: Your `replicationFactor` value cannot be more than the number of Kafka brokers.</li> <li> external_secret_name: To avoid exposing trustStorePassword, keyStorePassword, and jaasConfig parameters, leave the values empty and configure them using an External Secrets Manager, making sure you configure the keys in the secret in the order: STREAM_TRUSTSTORE_PASSWORD, STREAM_KEYSTORE_PASSWORD and STREAM_JAAS_CONFIG.
Enter the external secret name here. </li></ul> |